.expeditor/verify.pipeline.yml

```diff
@@ -16,3 +21,12 @@ steps:
     executor:
       docker:
         image: ruby:2.7-buster
+
+- label: run-tests-ruby-3.0
+  command:
+    - CI_ENABLE_COVERAGE=1 /workdir/.expeditor/buildkite/verify.sh
+  expeditor:
+    secrets: true
```

Comment:
We can't turn on secrets on chef-oss (the public pipeline). We need to remove this.